adapters/blip_2/dataloop.json

```diff
@@ -0,0 +1,114 @@
+{
+    "name": "blip-2-huggingface-app",
+    "description": "BLIP-2: Bootstrapping Language-Image Pre-training with Frozen Image Encoders and Large Language Models",
+    "attributes": {
+      "Provider": "Salesforce",
+      "Deployed By": "Dataloop",
+      "License": "BSD-3-Clause",
```

Comment:
the hugging face repo says this is under an MIT license. where did bsd3 come from?

Reply:
Copied the BLIP 1 adapter JSON, the license has changed for BLIP-2, nice catch!